src/plfit.h

```diff
@@ -22,7 +22,7 @@
 #define __PLFIT_H__
 
 #include <stdlib.h>
-#include "plfit_mt.h"
```

Comment:
Why is this change necessary? Wouldn't it work simply with the original double-quotes?